framework/scripts/agent_groups.py

```diff
@@ -36,138 +45,258 @@ def signal_handler(n_signal, frame):
 
 
 def show_groups():
-    groups_data = agent.get_agent_groups().to_dict()
-    print("Groups ({0}):".format(groups_data['total_affected_items']))
-    for g in groups_data['affected_items']:
-        print("  {0} ({1})".format(g['name'], g['count']))
+    """Show all the groups and the number of agents that belong to each one."""
+    groups = send_command(function=agent.get_agent_groups, command={})
+    unassigned_agents = send_command(function=agent.get_agents,
+                                     command={'q': 'id!=000;group=null'}).total_affected_items
 
-    print("Unassigned agents: {0}.".format(agent.get_agents(q='id!=000;group=null').to_dict()['total_affected_items']))
+    print(f"Groups ({groups.total_affected_items}):")
+    for items in groups.affected_items:
+        print(f"  {items['name']} ({items['count']})")
+
+    print(f"Unassigned agens: {unassigned_agents}.")
 
 
 def show_group(agent_id):
-    agent_info = agent.get_agents(agent_list=[agent_id]).to_dict()
-    if agent_info['total_affected_items'] == 0:
-        print(list(agent_info['failed_items'].keys())[0])
+    """Show the groups an agent belong to.
+
+    Parameters
+    ----------
+    agent_id: str
```

Comment:
Remember to leave a space after the variable names in the docstrings
```suggestion
    agent_id : str
```